_layouts/cv.html

```diff
@@ -0,0 +1,119 @@
+---
+layout: default
+---
+<!-- _layouts/cv.html -->
```

Comment:
this is a great page layout! thank you for contributing it. I noticed that you had to use a long list of if-then-else statements to determine the sub-layout block based on the entry type. I suggest splitting these sub-layouts into separate files that can be included, and keep these sub-components of the cv in `_includes/cv/*`. imo, this would improve readability and maintainability of this code. but what do you think?

(btw, some other long-ish layouts, such as `bib.html`, could benefit from the same refactor.)